stacks/container-apps/root.yml

```diff
@@ -573,47 +573,47 @@ Resources:
-#          Fn::ImportValue:
-#            !Sub "${InfrastructureNotificationsStackName}-OpsDebugMessagesSnsTopicArn"
-#        OpsErrorMessagesSnsTopicArn:
-#          Fn::ImportValue:
-#            !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
-#        EnvironmentType: !Ref EnvironmentType
-#        EnvironmentTypeAbbreviation: !Ref EnvironmentTypeAbbreviation
-#        EcrRegion: !Ref EcrRegion
-#        SecretsBase: !Ref SecretsBase
-#        ContainerMemory: !Ref ContainerMemory
-#        ContainerMemoryReservation: !Ref ContainerMemoryReservation
-#        ContainerCpu: !Ref ContainerCpu
-#        PlatformALBListenerPriorityPrefix: !Ref GrovePlatformALBListenerPriorityPrefix
-#        EcrImageTag: !Ref GroveEcrImageTag
-#        SecretsVersion: !Ref GroveSecretsVersion
-#        AppName: "grove"
-#        CreateWorker: "false"
-#        ContainerPort: "4204"
-#        HealthCheckPath: "/api/v1"
-#      Tags:
-#        - Key: "prx:cloudformation:stack-name"
-#          Value: !Ref AWS::StackName
-#        - Key: "prx:cloudformation:stack-id"
-#          Value: !Ref AWS::StackId
-#      TemplateURL: !Join ["", [!Ref TemplateUrlPrefix, "web-application.yml"]]
-#      TimeoutInMinutes: 5
+  GroveStack:
+    Type: "AWS::CloudFormation::Stack"
+    Condition: IsStaging
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        VPC: !Ref VPC
+        PlatformALBDNSName: !Ref PlatformALBDNSName
+        PlatformALBFullName: !Ref PlatformALBFullName
+        PlatformALBCanonicalHostedZoneID: !Ref PlatformALBCanonicalHostedZoneID
+        PlatformALBHTTPListenerArn: !Ref PlatformALBHTTPListenerArn
+        PlatformALBHTTPSListenerArn: !Ref PlatformALBHTTPSListenerArn
+        ECSCluster: !Ref ECSCluster
+        ECSServiceAutoscaleIAMRoleArn: !Ref ECSServiceAutoscaleIAMRoleArn
+        ECSServiceIAMRole: !Ref ECSServiceIAMRole
+        OpsDebugMessagesSnsTopicArn:
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsDebugMessagesSnsTopicArn"
+        OpsErrorMessagesSnsTopicArn:
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
+        EnvironmentType: !Ref EnvironmentType
+        EnvironmentTypeAbbreviation: !Ref EnvironmentTypeAbbreviation
+        EcrRegion: !Ref EcrRegion
+        SecretsBase: !Ref SecretsBase
+        ContainerMemory: !Ref ContainerMemory
+        ContainerMemoryReservation: !Ref ContainerMemoryReservation
+        ContainerCpu: !Ref ContainerCpu
+        PlatformALBListenerPriorityPrefix: !Ref GrovePlatformALBListenerPriorityPrefix
+        EcrImageTag: !Ref GroveEcrImageTag
+        SecretsVersion: !Ref GroveSecretsVersion
+        AppName: "grove"
+        CreateWorker: "false"
+        ContainerPort: "4204"
+        HealthCheckPath: "/api/v1"
```

Comment:
Does this path really exist for this app?  May just want `"/"` (or anything that returns a 200).

Reply:
nope, not sure what jingle is doing with it. _so many details_